File: WatchApp/ItemSuggestionCell.m
//
//  ItemSuggestionCell.m
//  WatchApp
//
//  Created by Stephane Bisson on 1/27/15.
//  Copyright (c) 2015 Stephane Bisson. All rights reserved.
//

#import "ItemSuggestionCell.h"
#import "SuggestionItem.h"

@implementation ItemSuggestionCell

- (void)updateWith:(id <TRSuggestionItem>)item {
    SuggestionItem *suggestionItem = (SuggestionItem*)item;
    self.textLabel.text = suggestionItem.completionText;
}

@end

File: WatchApp/ItemSuggestionCell.swift

import Foundation
import UIKit
import TRAutocompleteView

@objc class ItemSuggestionCell : UITableViewCell, TRAutocompletionCell {
    
    func updateWith(item: TRSuggestionItem!) {
        var suggestionItem:SuggestionItem = item as SuggestionItem
        
        self.textLabel!.text = suggestionItem.completionText()
    }
    
}